### DISCUSSION TITLE
[Scam Alert] ImToaken

### DISCUSSION THREAD

**Discussion by saeedkousha** [Webamoozcom/warning-list#120] [Announcements]
<div dir="rtl">

**Project Name:** 

ImToaken

**نام پروژه:** 

**نماد پروژه:**

**اتفاقی که افتاده است:**

پروژه ای که ادعا داره در مالزی فعالیت ماینینگ بیتکون انجام میدن و با تامین مالی جمعی به خرید ماینر مشغولن و روزانه تا 4 درصد سود از سرمایه گذاری شخصی و تا سه سطح هم سود ریفرال میدن اولین نکته قابل توجه اینه که این پروژه مالایی تمام کامیونیتیش فارسی هستن و در ایران به شدت در حال عضو گیری و جذب سرمایه است که باتوجه به کامیونیتی فارسی  بالا و ساز و کار شبکه ای این پروژه  در دسته پروژه های خطرناک قرار میگیرد و هشدار ریسک بالا دارد.
متاسفانه این قبیل پروژه ها خیلی زیاد شدن و با وعده های خوب کسب درآمد جیب خیلی ها رو خالی کردن که فقط داستان هایی که برای چگونگی درآمد شرکت تعریف میکنن متفاوت هست.
که جالبه خود بلاگر ها و معرف های این پروژه هیچ ضمانتی برای بقای این پروژه و انجام تعهدات آن ندارن و ریسک آن را به قول خودشان پذیرفتن

![imtoaken](https://user-images.githubusercontent.com/91057696/148898074-996e1ed3-611e-4e3b-ac74-f6c4100a15ee.png)

**قوانینی که نقض شده است:**

**نوع هشدار:**

هشدار ریسک بالا

**نوع کلاهبرداری:** 

پانزی

**وضعیت کلاهبرداری:**

در جریان

**نام کلاهبرداران:**

**نام شرکت(ها):**
new century
**آدرس وب سایت(ها):**
[imtoaken](https://www.imtoaken.cc/app/index.php?i=1&c=entry&m=ewei_shopv2&do=mobile&r=account.register#)

**برآورد مبلغ کلاهبرداری:**

**منابع:**
[imtoaken Telegram](https://t.me/imtoakentron)
[imtoaken Instagram](https://www.instagram.com/imtoaken.cc/)
[معرفی توسط مستر دیجی ارز](https://www.youtube.com/watch?v=S8zINakcUPc)

**جمع بندی:**

**Comment by Tecmiladro93** [comment]
ما یساله دنبال این آدمیم که میدونیم ایرانیه اونوقت تو با یه مقاله اشغال باعث شدی پروفایلاشو پاک کنه از سایت  خدا ازت نگذره

**Comment by Tecmiladro93** [comment]
اگه میشناسیش بگو حداقل پولای خودمونو برگردونه تو که فراریش دادی حداقل یکم مرد باش
